fairlearn/reductions/_moments/__init__.py

```diff
@@ -7,7 +7,7 @@
 from .moment import ClassificationMoment, LossMoment  # noq: F401
 
 from .conditional_selection_rate import ConditionalSelectionRate  # noqa: F401
-from .conditional_selection_rate import DemographicParity, EqualizedOdds  # noqa: F401
+from .conditional_selection_rate import DemographicParity, EqualizedOdds, EqualOpportunity  # noqa: F401, E501
 from .error_rate import ErrorRate   # noqa: F401
```

Comment:
We want a different name. Please review and incorporate my comments from the previous PR #325